packages/yoroi-extension/app/ergo-connector/stores/ConnectorStore.js

```diff
@@ -156,6 +156,7 @@ export default class ConnectorStore extends Store<StoresMap, ActionsMap> {
       pw: password,
     };
     window.chrome.runtime.sendMessage(sendData);
+    this._closeWindow();
```

Comment:
@rooooooooob the popup didn't close when you pressed "send" for the tx sign dialog. Was there a reason for this, or did we just accidentally omit this line?
Mostly I'm concerned with whether or not closing this page caused some in-background computation to abruptly halt.

Reply:
I don´t think there is unless @paulclindo knows something I don´t. I remember telling either you or Paul about this thinking it was just it forgetting to be closed somewhere. I recall it closing before, but that might've been just me remembering the old UI.

Reply:
Yeah, that should alright. I forgot to close it after confirming by mistake